repo/packages/E/elastic/5/resource.json

```diff
@@ -5,7 +5,12 @@
       "libmesos-bundle-tar-gz": "https://downloads.mesosphere.com/libmesos-bundle/libmesos-bundle-1.9.0-rc2-1.2.0-rc2-1.tar.gz",
       "executor-zip": "https://downloads.mesosphere.com/elastic/assets/1.0.8-5.2.2/executor.zip",
       "bootstrap-zip": "https://downloads.mesosphere.com/elastic/assets/1.0.8-5.2.2/bootstrap.zip",
-      "scheduler-zip": "https://downloads.mesosphere.com/elastic/assets/1.0.8-5.2.2/elastic-scheduler.zip"
+      "scheduler-zip": "file:///home/andrey/Projects/Ruv/dcos-commons/frameworks/elastic/build/distributions/elastic-scheduler.zip",
```

Comment:
Please bewere this is a local copy of the file. But since we needed to change also the elastic-scheduler to be capable of running offline, these changes where needed.